poc_mobile_verifier/src/owner_shares.rs

```diff
@@ -240,21 +315,15 @@ mod test {
         speedtests.insert(g2, VecDeque::from(g2_speedtests));
         let speedtest_avgs = SpeedtestAverages { speedtests };
 
-        let owner_rewards: HashMap<PublicKey, _> = SubnetworkRewards::from_epoch(
-            resolver,
-            &(now..(now + Duration::hours(24))),
-            heartbeats,
-            speedtest_avgs,
-        )
-        .await
-        .expect("Could not generate rewards")
-        .rewards
-        .into_iter()
-        .map(|p| (PublicKey::try_from(p.account).unwrap(), p.amount))
-        .collect();
+        let owner_rewards = OwnerShares::aggregate(&mut resolver, heartbeats, speedtest_avgs)
+            .await
+            .expect("Could not generate rewards");
 
         // The owner with two hotspots gets more rewards
-        assert!(owner_rewards.get(&owner1).unwrap() > owner_rewards.get(&owner2).unwrap());
+        assert!(
+            owner_rewards.shares.get(&owner1).unwrap().total_shares()
```

Comment:
can you replace these with expects to make the test clearer when it fails?

Reply:
yup